docs/docs/dagster-basics-tutorial/6-custom-components.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
I think it is worth worth mentioning this is doc that lots of integrations dynamically build definitions, which is what components were designed to do.